crates/energy/src/battery/component.rs

```diff
@@ -0,0 +1,112 @@
+use bevy::prelude::*;
+
+use crate::battery::{MIN_ATTACK_ENERGY, MIN_MOVE_ENERGY};
+
+/// The battery component is used to store the energy level of an entity.
+///
+/// # fields
```

Comment:
Rust's standard way of documenting arguments used throughout the repository is this:

```rust
/// # Arguments
///
/// * `capacity` - ...
///
/// * `energy` - ...
```

I.e. different title, empty lines and stars for bullet points.